bundles/tracker/donation/components/Donate.tsx

```diff
@@ -169,7 +170,8 @@ const Donate = (props: DonateProps) => {
               key={amountPreset}
               look={Button.Looks.OUTLINED}
               onClick={updateAmountPreset(amountPreset)}>
-              ${amountPreset}
+              {currencySymbol}
+              {amountPreset}
```

Comment:
With all of the other changes, this can now just be:
```suggestion
              {asCurrency(amountPreset, {currency})}
```

But you'll need to add `currency` as a destructured prop from `eventDetails` up on line 69 above.